### PR TITLE
add buttons and badge component in doc page

### PR DESCRIPTION
**Pull request by Ayushi20-19**
Add badge and button components in doc page with code link, there are three variant of the badge and four main and their subcategories of the buttons.
(some snaps attack below)

image
image
image
image

### CODE REVIEW

File: components/buttons.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  color: var(--black-color);
+}
+.btn-icon {
+  background-color: transparent;
+  border: 0;
+  margin: 1rem 1rem 1rem 2rem;
+}
+.floating-btn {
+  z-index: 9;
+}
```
Comment (starver20) [inline components/buttons.css]: Give some shadow, so it looks like the button is floating.

Comment (Ayushi20-19) [reply components/buttons.css]: okay

Comment (Ayushi20-19) [reply components/buttons.css]: done added the box-shadow

**Review by starver20 — APPROVED** [review]
Good to go.👍🏻

File: doc.html [hunk 2]
```diff
@@ -213,8 +215,194 @@ <h1>Alert</h1>
...
+         </div>
+        
+          <div class="code_link">
+             <code>
+              <iframe
+              src="https://carbon.now.sh/embed?bg=rgba%2867%2C196%2C255%2C1%29&t=one-light&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253C%21--primary%2520floating%2520button--%253E%250A%253Ca%2520href%253D%2522%2523%2522%253E%250A%2520%2520%2520%253Cbutton%2520class%253D%2522btn%2520primary%2520floating-btn%2522%253E%250A%2520%2520%2520%2520%253Ci%2520class%253D%2522fas%2520fa-arrow-up%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%253C%252Fbutton%253E%250A%2520%253C%252Fa%253E%250A%253C%21--outline%2520floating%2520button--%253E%250A%2520%253Ca%2520href%253D%2522%2523%2522%253E%250A%2520%2520%253Cbutton%2520class%253D%2522btn%2520primary-outline%2520floating-btn%2522%253E%250A%2520%2520%2520%253Ci%2520class%253D%2522fas%2520fa-arrow-up%2522%253E%253C%252Fi%253E%250A%2520%2520%253C%252Fbutton%253E%250A%253C%252Fa%253E"
+              style="width:100%; height: 100%; border:0; transform: scale(1); overflow:hidden;"
```
Comment (mohammed-sas) [inline doc.html:395]: i would suggest to avoid inline styling for iframe

**Review by mohammed-sas — APPROVED** [review]
other than suggested , pr is good to merge

File: doc.html (hunk 2 above)
Comment (Ayushi20-19) [reply doc.html:395]: okay I will create a class for it btw it just gives default so I change only width and height According to my code

**Pull request merged**